test/remote/gateways/remote_redsys_rest_test.rb

```diff
@@ -109,6 +110,13 @@ def test_successful_verify
     assert_equal 'Transaction Approved', response.message
   end
 
+  def test_successful_verify_without_cvv
```

Comment:
I opted for `verify` because our purchase test is currently failing due to required 3DS authentication.

I was able to confirm that the `SIS0217` error was thrown for this test before the conditional CVV logic was added, and now it succeeds.